src/main/kotlin/no/nav/familie/ef/sak/vilkår/gjenbruk/GjenbrukVilkårService.kt

```diff
@@ -54,32 +54,66 @@ class GjenbrukVilkårService(
             behandlingSomSkalOppdateres,
             behandlingIdSomSkalGjenbrukeInngangsvilkår,
         )
+        val vilkårsVurderingerForGjenbruk =
+            utledGjenbrukbareVilkårsvurderinger(
+                behandlingSomSkalOppdateres,
+                behandlingIdSomSkalGjenbrukeInngangsvilkår,
+            )
+        secureLogger.info(
+            "${SikkerhetContext.hentSaksbehandlerEllerSystembruker()} gjenbruker vurderinger fra behandling $behandlingIdSomSkalGjenbrukeInngangsvilkår " +
+                "for å oppdatere vurderinger på inngangsvilkår for behandling $behandlingSomSkalOppdateres",
+        )
+        vilkårsvurderingRepository.updateAll(vilkårsVurderingerForGjenbruk)
+    }
+
+    @Transactional
+    fun gjenbrukInngangsvilkårVurdering(
+        behandlingId: UUID,
```

Comment:
Hadde vært fint med konsekvent navngiving på argumentene. I denne funksjonen heter de for eksempel `behandlingId` og `behandlingIdForGjenbruk`. Så passes de videre inn i metoden `utledGjenbrukbareVilkårsvurderinger` hvor begge argumentene får nye navn.

Reply:
Enig, bytter til behandlingId og behandlingIdForGjenbruk, bedre med konsekvent  navngivning.